frodo.js

```diff
@@ -5,10 +5,10 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.